etna/core/static/css/wagtail-overrides.css

```diff
@@ -2,13 +2,45 @@
   font-size: 11px;
 }
 
+/* Add padding and background colour to Draftail character counts to
+aid legibility when appearing over help text */
+
+.inputlengthindicator {
+  display: block;
+  background-color: #fff;
+  position: absolute;
+  bottom: -1.6rem;
+  right: 0;
```

Comment:
I see your point about trying to make this ledgible, on mobile it's a displaying on-top of the help-text. Without some real reaching I don't think we can do too much about this. It's still ledgible but not an accessible, but then again an the original implementation I don't think this is solved either as it relies on not having help-text.

We could add a media query to the help-text to force the text to wrap at say 80% of the div size on mobile and 90% on desktop. I'll leave this to you to decide. 

TLDR; I think this is fine as is.

Reply:
Yes, It's a tricky one. I did initially use a `float: right` instead of absolute positioning, which helped somewhat, but felt a little more fragile. I'm probably just going to leave this as it is, as I want to avoid overriding default Wagtail stuff as much as possible.